recipes-core/packagegroups/packagegroup-ni-base.bb

```diff
@@ -48,7 +48,7 @@ RDEPENDS_${PN} += "\
 	eudev \
 	fw-printenv \
 	glibc-gconv-utf-16 \
-	gptfdisk-sgdisk \
```

Comment:
How much space does it save?

Reply:
`cgdisk` = 196 KiB
`gdisk` = 208 KiB
`fixparts` = 72 KiB
`sgdisk` = 196 KiB

So previously, we were just installing `sgdisk` at 196 KiB, now we would install all utilities at 672 KiB.